BUILDING.md

```diff
@@ -1,55 +1,14 @@
-# Warp 10 IDE Setup & Build  
+# Warp 10 Setup & Build  
 
-## Install Thrift on MAC OS
+## Install Thrift
+You need Thrift in order to build the Warp 10 platform.
+Currently, Warp 10 use Thrift 0.11.0
 
-The current version of thrift used by Warp 10 is thrift 0.11.0
-  
-Then add "/opt/thrift-0.11.0/bin" to your path and add `export THRIFT_HOME=/opt/thrift-0.11.0`
+Install it and then add "/opt/thrift-0.11.0/bin" to your path and add `export THRIFT_HOME=/opt/thrift-0.11.0`
```

Comment:
You just have to make sure `thrift` in in your path or you can define `THRIFT_HOME` if you don't want to. No need to do both. 

Also, the install may not be in `/opt`.

Reply:
done